enhancements/baremetal/baremetal-config-raid-and-bios.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Config-RAID-and-BIOS-for-Baremetal-IPI-deployments
+authors:
+  - "@fenggw-fnst"
+  - "@Hellcatlk"
+  - "@hs0210"
+  - "@zhouhao3"
+  - "@zhu1fei"
+reviewers:
+  - "@dtantsur"
+  - "@hardys"
+approvers:
+  -
+creation-date: 2021-05-18
+last-updated: 2021-08-23
+status: implementable
+see-also:
+  -
+replaces:
+  -
+superseded-by:
+  -
+---
+
+# Config RAID and BIOS for Baremetal IPI deployments
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add the support of RAID and BIOS configuration for baremetal IPI deployments.
+
+## Motivation
+
+With the new features of [RAID configuration](https://github.com/metal3-io/baremetal-operator/pull/292) and [BIOS configuration](https://github.com/metal3-io/baremetal-operator/pull/302), openshift already has the ability to configure RAID and BIOS for baremetal.
+
+However, these features only work with a running cluster, users cannot configure desired RAID and BIOS during IPI deployments.
+
+### Goals
```

Comment:
Will it be applicable for adding nodes as well?
What about ZTP support?

Reply:
> Will it be applicable for adding nodes as well?

The latest OCP version already supports configuring RAID and BIOS when adding worker nodes. And this proposal is dedicated to supporting the RAID and BIOS configuration of the master node during the IPI process.

> What about ZTP support?

This proposal have nothing to do with ZTP.